packages/block-library/src/columns/edit.js

```diff
@@ -82,15 +82,18 @@ export const ColumnsEdit = function( { attributes, setAttributes, className, upd
 	);
 };
 
+const DEFAULT_EMPTY_ARRAY = [];
+
 export default compose(
 	/**
 	 * Selects the child column Blocks for this parent Column
 	 */
 	withSelect( ( select, { clientId } ) => {
 		const { getBlocksByClientId } = select( 'core/editor' );
+		const block = getBlocksByClientId( clientId )[ 0 ];
```

Comment:
Out of curiosity, I'm wondering if the following would work as well:
```js
const { innerBlocks = DEFAULT_EMPTY_ARRAY } = getBlocksByClientId( clientId )[ 0 ];
```

Reply:
That won't work if the array returned by `getBlocksByClientId` is empty.  It'll result in something like:
>  "Can't destructure property 'innerBlocks' of 'undefined' or 'null'"